roles/kubernetes-apps/ingress_controller/meta/main.yml

```diff
@@ -4,25 +4,26 @@ dependencies:
     when: ingress_nginx_enabled
     tags:
       - apps
-      - ingress-nginx
       - ingress-controller
+      - ingress-nginx
 
   - role: kubernetes-apps/ingress_controller/ambassador
     when: ingress_ambassador_enabled
     tags:
       - apps
-      - ambassador
       - ingress-controller
+      - ambassador
 
   - role: kubernetes-apps/ingress_controller/cert_manager
     when: cert_manager_enabled
     tags:
       - apps
-      - cert-manager
       - ingress-controller
+      - cert-manager
 
   - role: kubernetes-apps/ingress_controller/alb_ingress_controller
     when: ingress_alb_enabled
     tags:
       - apps
+      - ingress-controller
       - ingress_alb
```

Comment:
This part comes from a big PR https://github.com/kubernetes-sigs/kubespray/pull/5489
This seems a valid change for me.